src/main/java/de/chojo/repbot/service/ReputationService.java

```diff
@@ -89,6 +89,9 @@ public boolean submitReputation(Guild guild, User donor, User receiver, Message
             case REACTION -> {
                 if (!settings.isReactionActive()) return false;
             }
+            case EMBED -> {
+                //TODO: allow enabling and disable embed reputation?
```

Comment:
Are you still thinking about it or will that be part of another issue, if you decide to do that?

Reply:
I am thinking of it. might create an issue if I make an decision. Just wanted to have an reminder without any commitment.